### PR TITLE
feat(3171): Move pipeline template workflowGraph out of config into a new field

### PR DESCRIPTION
**Pull request by sagar1312**
BREAKING CHANGE: Move workflowGraph out of config and keep it at the same level as config

## Context

Changes were made in https://github.com/screwdriver-cd/data-schema/pull/571 (and related PRs) to store `workflowGraph` of pipeline template as part of the `config

## Objective

Move `workflowGraph` out of `config` and keep it at the same level as `config`

## References

https://github.com/screwdriver-cd/screwdriver/issues/3171

## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,10 +102,10 @@
     "screwdriver-build-bookend": "^4.0.0",
     "screwdriver-cache-bookend": "^3.0.0",
     "screwdriver-command-validator": "^3.0.0",
-    "screwdriver-config-parser": "^10.4.1",
+    "screwdriver-config-parser": "file:../config-parser/screwdriver-config-parser-10.0.0.tgz",
```
Comment (sagar1312) [inline package.json:105]: TODO: Update the version after screwdriver-cd/config-parser#170 is merged

File: package.json [hunk 2]
```diff
@@ -102,10 +102,10 @@
...
     "screwdriver-coverage-bookend": "^2.0.0",
     "screwdriver-coverage-sonar": "^4.1.1",
-    "screwdriver-data-schema": "^23.7.0",
+    "screwdriver-data-schema": "file:../data-schema/screwdriver-data-schema-22.0.0.tgz",
```
Comment (sagar1312) [inline package.json:108]: TODO: Update the version after screwdriver-cd/data-schema#575 is merged

File: package.json [hunk 3]
```diff
@@ -114,7 +114,7 @@
     "screwdriver-executor-queue": "^4.0.0",
     "screwdriver-executor-router": "^3.0.0",
     "screwdriver-logger": "^2.0.0",
-    "screwdriver-models": "^29.18.4",
+    "screwdriver-models": "file:../models/screwdriver-models-29.0.0.tgz",
```
Comment (sagar1312) [inline package.json:117]: TODO: Update the version after https://github.com/screwdriver-cd/models/pull/627 is merged

File: package.json [hunk 4]
```diff
@@ -123,8 +123,8 @@
     "screwdriver-scm-github": "^12.6.0",
     "screwdriver-scm-gitlab": "^3.1.0",
     "screwdriver-scm-router": "^7.1.0",
-    "screwdriver-template-validator": "^8.2.0",
-    "screwdriver-workflow-parser": "^4.4.1",
+    "screwdriver-template-validator": "file:../template-validator/screwdriver-template-validator-8.0.0.tgz",
```
Comment (sagar1312) [inline package.json:126]: TODO: Update the version after https://github.com/screwdriver-cd/template-validator/pull/42 is merged

File: package.json [hunk 5]
```diff
@@ -123,8 +123,8 @@
...
+    "screwdriver-workflow-parser": "file:../workflow-parser/screwdriver-workflow-parser-4.0.0.tgz",
```
Comment (sagar1312) [inline package.json:127]: TODO: Update version after screwdriver-cd/workflow-parser#55 is merged

**Review by minghay — APPROVED** [review]

**Review by VonnyJap — APPROVED** [review]

**Pull request merged**